src/app/performance-list/_components/performance-tabs.tsx

```
@@ -54,29 +68,37 @@ export function PerformanceTabs({ performances = [] }: PerformanceTabsProps) {
-            <UpcomingList performances={performances} />
+            <Suspense fallback={<div className="flex justify-center py-20" role="status">로딩 중...</div>}>
+              {isSearchMode
+                ? <PerformanceSearchMode keyword={searchKeyword} validTab="upcoming" />
+                : <PerformanceListMode validTab="upcoming" />}
+            </Suspense>
           </TabsContent>
           <TabsContent value="past">
-            <PastList />
+            <Suspense fallback={<div className="flex justify-center py-20" role="status">로딩 중...</div>}>
+              {isSearchMode
+                ? <PerformanceSearchMode keyword={searchKeyword} validTab="upcoming" />
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)